kernelci.org/content/en/docs/contrib.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributing Guidelines"
+date: 2022-12-08T09:21:56Z
+draft: false
+weight: 2
+---
+
+KernelCI core project is open for contributions. Contributions may consist of
+adding new build, tests and device types as well as features and bugfixes for
+KernelCI core tools.
+The best way to contribute is to send a PR to [kernelci-core](https://github.com/kernelci/kernelci-core).
+When the PR is created, [KernelCI staging](https://kernelci.org/docs/instances/staging)
+instance takes care of updating the [staging.kernelci.org branch](https://github.com/kernelci/kernelci-core/tree/staging.kernelci.org).
+In general the branch is updated every 8h and a limited set of builds and tests
+is run on it. More detailed information about the logic behind staging runs can
```

Comment:
```suggestion
are run on it. More detailed information about the logic behind staging runs can
```

Reply:
I'm not sure if it should be **are** I'm mentioning a _set_ here, that's why I used **is**.  Having said that I'm not a native English speaker so I might be wrong. @gctucker maybe you could help?

Reply:
Even if the noun is singular, when it designates a plural entity it should be treated as plural e.g. "the team are on track to win the championship".  So it should be "are" imho.

Reply:
I think either is fine, as far as I understand it depends on what one wants to emphasize, and singular is more common in American English (https://en.wikipedia.org/wiki/American_and_British_English_grammatical_differences#Subject-verb_agreement).